pkg/cmd/util/output/backup_describer.go

```diff
@@ -467,3 +479,40 @@ func (v *volumesByPod) Sorted() []*podVolumeGroup {
 
 	return v.volumesByPodSlice
 }
+
+func DescribeCSIVolumeSnapshots(d *Describer, details bool, volumeSnapshotContents []snapshotv1beta1api.VolumeSnapshotContent) {
+	if !features.IsEnabled(velerov1api.CSIFeatureFlag) {
+		return
+	}
+
+	if !details {
+		d.Printf("CSI Volume Snapshots:\t%d included (specify --details for more information)\n", len(volumeSnapshotContents))
+		return
+	}
+	d.Printf("CSI Volume Snapshots:\n")
+
+	for _, vsc := range volumeSnapshotContents {
+		DescribeVSC(d, details, vsc)
+	}
+}
+
+func DescribeVSC(d *Describer, details bool, vsc snapshotv1beta1api.VolumeSnapshotContent) {
+	if vsc.Status == nil {
+		d.Printf("Volume Snapshot Content %s cannot be described because its status is nil\n", vsc.Name)
+		return
+	}
+
+	d.Printf("Snapshot Content ID: %s\n", vsc.Name)
```

Comment:
any particular reason to call it "ID" rather than "Name"?

Reply:
No, not really. Updating.